react/src/views/Transfers/ShipmentView/components/ShipmentCard.tsx

```diff
@@ -103,7 +103,7 @@ function ShipmentCard({ shipment }: IShipmentProps) {
         <StackDivider borderColor="blackAlpha.800" marginTop={-1.5} />
         <Box p={4}>
           <Center alignContent="stretch">
-            <Text fontWeight="bold">TOTAL:</Text> {shipment.details.length} boxes
+            <Text fontWeight="bold">TOTAL:</Text> {shipment.details?.length!} boxes
```

Comment:
@vahidbazzaz test 4.5.1.4 was failing due to a typeError here.

Reply:
@HaGuesto I tested it locally, and the mock data seemed to work fine